<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="uiDescriptorsTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>1121</width>
     <height>801</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>change_fp_type()</slot>
 </slots>
</ui>
